server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -33,38 +33,33 @@
 @Singleton
 public class RwServer extends Thread {
 
-    private static final Logger      LOG                         = LoggerFactory.getLogger(RwServer.class);
-    private static final int         COMPRESSION_THRESHOLD_BYTES = 1000;
-    private static final Set<String> COMPRESSIBLE_MIME_TYPES      = new HashSet<>(asList(
+    private static final Logger LOG = LoggerFactory.getLogger(RwServer.class);
+    private static final int COMPRESSION_THRESHOLD_BYTES = 1000;
+    private static final Set<String> COMPRESSIBLE_MIME_TYPES = new HashSet<>(asList(
         "text/plain",
         "application/xml",
         "text/css",
         "application/x-javascript",
         "application/json"
     ));
+    private static final long SECONDS_WAIT_BEFORE_SHUTDOWN = 5;
```

Comment:
Should we add this to the ServerConfig instead so that we can make configuration changes if it would be necessary ?